ext/spl/spl_dllist.stub.php

```diff
@@ -57,20 +47,19 @@ public function getIteratorMode() {}
     public function offsetExists($index) {}
```

Comment:
Wanted to say that these should have an `int` type now, but ... gah, we can't :(